### ISSUE TITLE
Setup as Firefox extension

### ISSUE THREAD

**Issue by moollaza** [oslabs-beta/svelte-sight#58]
Firefox and Chrome apps share most of the same APIs and can fairly easily share a single code base. It would be great if this was available to Firefox devs as well!

**Comment by ghost** [comment]
Agreed. A lot of devs use Firefox due to the privacy that comes with it, and its lifted developer restrictions to certain APIs that could limit ads, which provides more developer freedom.
I'd also like to see a few tweaks to make it available for Firefox.

**Comment by alpap** [comment]
True it would be nice to see it for firefox
